lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -128,8 +129,8 @@ class ScreenshotThumbnails extends Audit {
       }
       thumbnails.push({
         timing: Math.round(targetTimestamp - speedline.beginning),
-        timestamp: targetTimestamp * 1000,
-        data: base64Data,
+        timestamp: targetTimestamp,
+        data: `data:image/jpeg;base64,${base64Data}`,
```

Comment:
yeah, having the prefix makes sense so no one has to guess what type of image it is, even if it's annoying to add it here